torch/_inductor/pattern_matcher.py

```diff
@@ -308,7 +321,7 @@ def __repr__(self):
         return f"{self.__class__.__name__}({self.fns_repr()})"
 
     def has_multiple_users(self) -> bool:
-        return self.users is MULTIPLE or self.users > 1
+        return isinstance(self.users, Multiple) or self.users > 1
```

Comment:
This is less clear `isinstance(self.users, Multiple)` than what existed previously `self.users is MULTIPLE`

Reply:
Unfortunately, I don't see how to get this to typecheck with mypy other than this.